ArduPlane/ArduPlane.cpp

```diff
@@ -945,4 +945,11 @@ void Plane::publish_osd_info()
 }
 #endif
 
+void Plane::set_motor_emergency_stop(bool b)
+{
+    if(motor_emergency_stop != b) {
```

Comment:
Just do the assignment?

Reply:
... also, space after the `if`

Reply:
Your right, I was looking at this when I was doing the re-base. I just coped the function direct from copter, I thought there must have been some reason to do it this way.